includes/oauth/class-google-oauth.php

```diff
@@ -339,17 +338,34 @@ private static function authenticated_user_basic_information() {
 			);
 			if ( 200 === wp_remote_retrieve_response_code( $user_info_response ) ) {
 				$user_info = json_decode( $user_info_response['body'] );
-				return [
-					'email'             => $user_info->email,
-					'has_refresh_token' => null !== $oauth2_credentials->getRefreshToken(),
-				];
+				return $user_info->email;
 			}
 		} else {
 			Logger::log( 'Failed retrieving user info – invalid credentials.' );
 			return new \WP_Error( 'newspack_google_oauth', __( 'Invalid Google credentials. Please reconnect.', 'newspack' ) );
 		}
+	}
+
+	/**
+	 * Authenticated user's basic information.
+	 *
+	 * @return object|WP_Error Basic information, or error.
```

Comment:
```suggestion
	 * @return array|WP_Error Basic information, or error.
```

This method also returns `false`, returning an empty array would be a better practice to avoid type errors in the future.